template/client/views/Home/index.js

```diff
@@ -18,9 +20,11 @@ export default {
 
     return (
       <div class='page'>
+        <Title level='h1'> H1 Title</Title>
         <p>
           To get started, edit files in <code>./client</code> and save.
         </p>
+        <Title level='h2' highlighted>Modified H2 Title</Title>
```

Comment:
Would it make sense to provide an array like `<Title level='h2' modifiers=['highlighted']>Modified H2 Title</Title>`? Dunno if it's proper syntax…